.gitignore

```
@@ -16,3 +16,4 @@ exchanges/*/LICENSE
 # TODO: Figure out how to remove these:
 tmp/
 dist/
+examples/react-urql-query/public/*
```

is it worth generalising this to the following for the other `examples` that are about to be added?

```suggestion
examples/*/public
```

On another small note, I believe you've used `npm` rather than `yarn` to install dependencies. That's not a problem, but what I wanted to point out is that no lockfile was added to the respository because we're ignoring `package-lock.json`. I'd actually avoid adding lockfiles in the future for each example too, because they're really not worth keeping up-to-date as little breakage should occur with the correct dependency ranges for simple example apps, so I'd also add:

```
examples/*/yarn.lock
```